apps/webshop/models.py

```diff
@@ -23,6 +23,22 @@ class Product(models.Model):
     deadline = models.DateTimeField(null=True, blank=True)
     active = models.BooleanField(default=True)
 
+    def calculate_stock(self):
+        if self.product_sizes.count() > 0:
+            return sum((product_size.stock for product_size in self.product_sizes.all()))
```

Comment:
It's really only necessary to wrap a generator expression in parenthesis if you are solely assigning it to a variable like `top = (kek.bur for kek in iterable)`